sectionctl.go

```diff
@@ -38,15 +39,25 @@ type CLI struct {
 	SectionAPIPrefix   *url.URL                     `default:"https://aperture.section.io" env:"SECTION_API_PREFIX"`
 	SectionAPITimeout  time.Duration                `default:"30s" env:"SECTION_API_TIMEOUT" help:"Request timeout for the Section API"`
 	InstallCompletions kongplete.InstallCompletions `cmd:"" help:"install shell completions"`
+	Quiet					     bool                         `env:"SECTION_CI" help:"Enables minimal logging, for use in continuous integration."`
 }
 
-func bootstrap(c CLI, ctx *kong.Context) {
+
+
+
+func bootstrap(c CLI, ctx *kong.Context) context.Context {
 	api.PrefixURI = c.SectionAPIPrefix
 	api.Timeout = c.SectionAPITimeout
 
+	contxt := context.Background()
```

Comment:
use `ctx` instead. Don't use the name of a package as variable. The auto-complete doesn't work after this line for the `context` package.

Reply:
`ctx` is used on line 112 for the Kong parsing context. I was hesitant to replace that, but I can rename it so I can use `ctx` as context.Context in every file